builtin/providers/openstack/resource_openstack_compute_secgroup_v2.go

```diff
@@ -122,7 +122,7 @@ func resourceComputeSecGroupV2Read(d *schema.ResourceData, meta interface{}) err
 	d.Set("name", sg.Name)
 	d.Set("description", sg.Description)
 	log.Printf("[DEBUG] rulesToMap(sg.Rules): %+v", rulesToMap(sg.Rules))
-	d.Set("rules", rulesToMap(sg.Rules))
+	d.Set("rule", rulesToMap(sg.Rules))
```

Comment:
I think it was also the ruleID key as well. I've noticed that if the map does not strictly adhere to the schema, no computed values will be rendered. Maybe this is just a coincidental behavior that I've seen, though.